src/cfvariable.jl

```diff
@@ -448,22 +448,30 @@ end
 #@inline CFtransformdata(data::Char,fv,scale_factor,add_offset,time_origin,time_factor,DTcast) = CFtransform_missing(data,fv)
 #@inline CFinvtransformdata(data::Char,fv,scale_factor,add_offset,time_origin,time_factor,DT) = CFtransform_replace_missing(data,fv)
 
-function Base.getindex(v::CFVariable, indexes::TIndices...)
+function DiskArrays.readblock!(v::CFVariable{T, N},
+    aout,
+    indexes::Vararg{OrdinalRange, N}) where {T, N}
+
     data = parent(v)[indexes...]
```

Comment:
Maybe there is a more performant way where we use `readblock!` on `parent(v)`

Reply:
Yes in this case you know the chunking is the same. But unfortunately the type may not be so you can't reuse the buffer array. 

If you know the parent array type you can allocate a new buffer and call readblock! with it